app/views/static_pages/home.html.erb

```diff
@@ -0,0 +1,7 @@
+<% provide(:title, "Home") %>
+<h1>Sample App</h1>
+<p>
+    This is the home page for the
+    <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
+    sample application.
+</p>
```

Comment:
Add a single new line at the end of files (https://github.com/airbnb/ruby#trailing-newline)

Reply:
I'd suggest setting your IDE to do this for you ;)